packages/compass-import-export/src/components/export-select-fields.tsx

```diff
@@ -0,0 +1,373 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import {
+  Body,
+  Button,
+  Checkbox,
+  Disclaimer,
+  ErrorSummary,
+  Icon,
+  Placeholder,
+  Table,
+  TableHeader,
+  Row,
+  Cell,
+  TextInput,
+  css,
+  spacing,
+  Label,
+} from '@mongodb-js/compass-components';
+import { connect } from 'react-redux';
+
+import {
+  getLabelForFieldId,
+  selectFieldsToExport,
+  toggleFieldToExport,
+  addFieldToExport,
+  toggleExportAllSelectedFields,
+  getIdForSchemaPath,
+} from '../modules/export';
+import type { FieldsToExport } from '../modules/export';
+import type { RootExportState } from '../stores/export-store';
+import type { SchemaPath } from '../export/gather-fields';
+
+const headerContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  marginBottom: spacing[3],
+  marginTop: spacing[2],
+  gap: spacing[2],
+});
+
+const tableContainerStyles = css({
+  maxHeight: spacing[7] * 3,
+  overflow: 'auto',
+});
+
+const smallCellContainerStyle = css({
+  width: spacing[5],
+  margin: '0 auto',
+});
+
+const textInputStyles = css({
+  padding: `${spacing[1]}px 0`,
+  minWidth: spacing[7] * 3,
+});
+
+const enterToAddStyles = css({
+  display: 'inline-flex',
+  alignItems: 'center',
+  marginLeft: spacing[2],
+});
+
+const placeholderStyles = css({
+  margin: spacing[1],
+});
+
+const retryButtonContainerStyles = css({
+  margin: spacing[2],
+  textAlign: 'center',
+});
+
+const addNewFieldRowStyles = css({
+  marginBottom: spacing[5],
+});
+
+const placeholdersCount = 6;
```

Comment:
What does this placeholder do?

Reply:
They are used to show the skeleton loaders on the page when the sample documents are being loaded. I updated the variable name to be `loadingPlaceholderCount` to be a bit more descriptive. 

<img width="686" alt="Screenshot 2023-04-18 at 8 22 50 AM" src="https://user-images.githubusercontent.com/1791149/232778518-2dc9025c-7480-4573-af69-784b16ba1e6e.png">